### PR TITLE
Setup Of Vet_clinics Database

### PR DESCRIPTION
**Pull request by gisachris**
## IN This PR I present the Setup Of the Vet_clinics Database:📦

Below are some of my implementations as well as going over the project requirements 🤝 .
## Requirements
- use GitFlow 🌊.
- Create a well-organized `Readme` file 📜.
- Add to the `schema.SQL` file with the required content. ⛱️
create_table
- Add to the `data.SQL` file with the required content. 🛰️
Insert_into_table
- Add to the `query.SQL` file with the required content. 🧭
> Query that selects name and escape attempts of animals that weigh more than 10.5kg
select_name_escape_by_weight
> Query that selects all animals not named Gabumon
select_name_not_gabumon
>Query that selects all animals that are neutered
select_name_where_neutered_true
>Query that selects name of all animals that are neutered and have less than 3 escape attempts.
select_name_with_nurture_attempts
>Query that selects all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg
select_all_between_weights
>Query that selects date of birth of all animals named either "Agumon" or "Pikachu"
select_dob_for_spec_names
>Query that selects the name of all animals born between 2016 and 2019
select_name_between_dates
>Query that selects all animals whose name ends in "mon"
select_name_ends

## Achievements
- use GitFlow 🌊 .✅
- Create a well-organized `Readme` file 📜.✅
- Add to the `schema.SQL` file with the required content. ⛱️✅

- Add to the `data.SQL` file with the required content. 🛰️✅
- Add to the `query.SQL` file with the required content. 🧭✅

## Conclusion
**We are looking forward to hearing 👂  from you Our CR in terms of what can be better implemented 👨‍💻  for the project thankyou!. And we look forward to hearing from you soon.**  🙇

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,7 @@
...
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Argon','2020-02-03',0,TRUE,'10.23');
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES('Gabumon','2018-11-15',2,TRUE,8.0),
+      ('Pickachu','2021-01-7',1,FALSE,15.04),
+      ('Devimon','2017-05-12',5,TRUE,11.00);
```
Comment (AlphaNtihinduka) [inline data.sql]: - [ ] As [the project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md#project-requirements) suggest you should also insert the values of `Agumon`

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @gisachris 🙋‍♂️
Good job working on vet clinic. You have done incredibly well. 👏  

You still need to implement few on the project but you are almost there!!! 🚶: 

TO HIGHLIGHT:
- Used queries.sql file to save queries that analyze data ✔️
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: data.sql (hunk 1 above)
Comment (gisachris) [reply data.sql]: Great insight @AlphaNtihinduka. I will get on this and fix it, thank you.

**Comment by gisachris** [comment]
## IN This PR I present the Requested ♻️ Changes Of the Vet_clinics Database: 📦

Below are some of my implementations as well as going over the project requirements 🤝 .
## Requirements
- Correctly insert all the elements in the Animals 🦁 Table 🗳️ 
Insert_into_table

## Achievements
- Correctly insert all the elements in the Animals 🦁 Table 🗳️ ✅

## N.B
I first used the `Truncate` command 🔴 to empty the table of the previous records  then inserted the new data to enforce Data Integrity.

## Conclusion
**We are looking forward to hearing 👂  from you Our CR in terms of what can be better implemented 👨‍💻  for the project thankyou!. And we look forward to hearing from you soon.**  🙇

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @gisachris 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Used queries.sql file to save queries that analyze data ✔️ 
- Well-structured files :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**